include/openPMD/Iteration.hpp

```diff
@@ -99,8 +136,33 @@ class Iteration : public Attributable
     void flush();
     void read();
 
+    enum class CloseStatus
```

Comment:
Please see https://github.com/openPMD/openPMD-api/blob/beea53937ec1b4774ab087ae24f529d8df0c4cc1/include/openPMD/IO/Access.hpp#L28-L35 for an enum class doxygen string.